bundles/storage/org.eclipse.smarthome.storage.json/src/main/java/org/eclipse/smarthome/storage/json/JsonStorage.java

```diff
@@ -203,10 +208,10 @@ private T deserialize(Map<String, Object> jsonValue) {
 
             FileReader reader = new FileReader(inputFile);
             Map<String, Map<String, Object>> type = new HashMap<String, Map<String, Object>>();
```

Comment:
No, it makes a difference. The type information actually _is_ available and gets used (see [%24Gson%24Types.java#L317](https://github.com/google/gson/blob/a300148003e3a067875b1444e8268b6e0f0e0e02/gson/src/main/java/com/google/gson/internal/%24Gson%24Types.java#L317))